src/meili/match_route.cc

```diff
@@ -208,69 +216,152 @@ bool MergeRoute(std::vector<EdgeSegment>& route, const State& source, const Stat
     throw std::logic_error("The first edge must be an origin (invalid predecessor)");
   }
 
-  MergeEdgeSegments(route, segments.rbegin(), segments.rend());
+  // TODO: why doesnt routing.cc return trivial routes? move this logic there
+  // Might be a trivial route
+  if (segments.empty()) {
+    return false; // this is a discontinuity for now
+    // If we have no chance of making a trivial route bail
```

Comment:
Asking only since we have other commented out code committed to the code base: are we planning on removing this block of comments after the return statement?